README.md

```diff
@@ -602,6 +602,37 @@ The response body includes:
 }
 ```
 
+## Public Metrics API
```

Comment:
A general naming nit: calling this "public metrics API" seems weird if the API isn't actually publicly accessible. And it might be more flexible not to imply that the metrics returned by this are always visible through some public UI, either.

You could just call it "MetricSets" or whatever.

Reply:
Yeah, I agree. My rationale here was that the structure of the response type is currently very specific to these kinds of single-dimensional broad aggregations, and I also wasn't sure about reconciling this against the existing metrics API.

I changed it to just MetricSets instead.